src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -166,10 +166,8 @@ public function getStyle()
      *
      * @return $this
      */
-    public function setColumnStyle($columnIndex, $name)
+    public function setColumnStyle(int $columnIndex, $name)
```

Comment:
The `@param` related to `$columnIndex` can be removed. Can you double-check if we can remove more?

Reply:
I've double checked all the classes, I think we don't have anyone left.